resources/conf.edn

```diff
@@ -14,7 +14,7 @@
              :pubkey #profile {:development "auth_pubkey.pem"
                                :test "test_pubkey.pem"
                                :production "/root/prod_pubkey.pem"}
-             :secrets #include "secrets.edn"
+             :secrets #include #join [#env HOME "/.secrets.edn"]
```

Comment:
Will the file always be in the root directory of the filesystem if not specified? Could be a silly question on my part.